stellargraph/data/explorer.py

```diff
@@ -102,12 +102,12 @@ class GraphWalk(object):
     Base class for exploring graphs.
     """
 
-    def __init__(self, graph, graph_schema=None, seed=None):
+    def __init__(self, graph, graph_schema=None, seed=None, use_ilocs=False):
```

Comment:
I made this a parameter to avoid breaking the API, but seeing as its mostly internal and all internal uses of these classes set `use_ilocs=True` it could probably be removed entirely - i.e. only support ilocs. What do you think?

Oh, and the iloc/ids gaurding is done in  `GraphWalk.neighbors`